core/station/impl/Cargo.toml

```diff
@@ -41,6 +41,10 @@ thiserror = { workspace = true }
 uuid = { workspace = true, features = ['v4'] }
 station-api = { path = '../api', version = '0.0.2-alpha.3' }
 upgrader-api = { path = '../../upgrader/api', version = '0.0.2-alpha.3' }
+alloy = { version = "0.2.0", features = ["signers", "k256", "consensus", "eips"] }
+alloy-chains = "0.1.23"
+evm-rpc-canister-types = "1.0.0"
+maplit = "1.0.2"
```

Comment:
TODO: remove this in favour of a bunch of `if-else` statements. Less deps -> less headache